packages/cron/src/jobs/metrics.js

```diff
@@ -85,6 +91,25 @@ async function updateUploadsCount(roPg, rwPg, type) {
   ])
 }
 
+/**
+ * @param {Client} roPg
+ * @param {Client} rwPg
+ */
+async function updateUploadTotal(roPg, rwPg) {
+  const { rows } = await roPg.query(COUNT_TOTAL_UPLOADS)
+  if (!rows.length)
+    throw new Error(`no rows returned counting past weeks uploads`)
+  await rwPg.query(UPDATE_METRIC, [`uploads_all_types_total`, rows[0].total])
```

Comment:
This is not used by the `/stats` route is it?

Reply:
It is not, i'll remove this change, I shuffled some things around, I'll make sure there's nothing there that needs changed.